analysis/seed_scanner.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Replace this with the flags you want to test.
+FLAGS="--approach oracle --env blocks --timeout 0.1"
+START_SEED=0
+NUM_SEEDS=100
+
+for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
+    if python src/main.py $FLAGS --num_test_tasks 1 --seed $SEED | grep -q 'Tasks solved: 0'
+    then
+        echo "Found failing seed: ${SEED}."
+        echo "Command to reproduce:"
+        echo "python src/main.py ${FLAGS} --num_test_tasks 1 --seed ${SEED}"
```

Comment:
to ensure this doesn't diverge from line 9 in the future, let's make a variable `CMD="python src/main.py ${FLAGS} --num_test_tasks 1 --seed ${SEED}"` at the start of each for loop iteration, then use that both in line 9 and here